linux/ubuntu/17.04/foxpass_setup.py

```diff
@@ -27,11 +27,13 @@
 import argparse
 from datetime import datetime
 import os
-import os.path
 import re
 import sys
 import urllib3
 
+from datetime import datetime
```

Comment:
this is already included 6 lines up.

Reply:
Put that there in the PEP8 cleanup and missed removing the previous call, have done so.  GitHub isn't picking up that change.

Reply:
huh, still hasn't come through.

Reply:
github says I pushed it 6 hours ago (e0791c8)

Reply:
oh i see. i was expecting you to remove the one above this comment string, not the one on top. datetime is a system lib, so it belongs in the top group.

Reply:
Ahh, for some reason I thought that sub-class imports went in their own section.